src/wasm-type.h

```diff
@@ -60,14 +64,19 @@ class Type {
   const std::vector<Type>& expand() const;
 
   // Predicates
-  bool isSingle() const { return id >= i32 && id < _last_value_type; }
-  bool isMulti() const { return id >= _last_value_type; }
-  bool isConcrete() const { return id >= i32; }
-  bool isInteger() const { return id == i32 || id == i64; }
-  bool isFloat() const { return id == f32 || id == f64; }
-  bool isVector() const { return id == v128; };
-  bool isNumber() const { return id >= i32 && id <= v128; }
-  bool isRef() const { return id >= funcref && id <= exnref; }
+  constexpr bool isSingle() const { return id >= i32 && id <= last_value_type; }
+  constexpr bool isMulti() const { return id > last_value_type; }
+  constexpr bool isConcrete() const { return id >= i32; }
+  constexpr bool isInteger() const { return id == i32 || id == i64; }
+  constexpr bool isFloat() const { return id == f32 || id == f64; }
+  constexpr bool isVector() const { return id == v128; };
+  constexpr bool isNumber() const { return id >= i32 && id <= v128; }
+  constexpr bool isRef() const { return id >= funcref && id <= exnref; }
+  constexpr uint32_t getID() const { return id; }
+  constexpr ValueType getVT() const {
+    assert(!isMulti() && "Unexpected multivalue type");
+    return static_cast<ValueType>(id);
+  }
```

Comment:
I think `getVT` is a little unclear. For consistency with `isSingle()`, how about `getSingle()`?

Reply:
Can do 👍